gramps/gui/dialog.py

```diff
@@ -506,6 +508,11 @@ def main(args):
     win = Gtk.Window()
     win.set_title('Dialog test window')
     win.set_position(Gtk.WindowPosition.CENTER)
+    #Set the mnemonic modifier on Macs to alt-ctrl so that it
+    #doesn't interfere with the extended keyboard, see
+    #https://gramps-project.org/bugs/view.php?id=6943
+    if(is_quartz()):
```

Comment:
C:514, 0: Unnecessary parens after 'if' keyword (superfluous-parens)

Reply:
Heh, my C++ bias is showing again...